main.py

```diff
@@ -112,6 +112,12 @@ def get(self):
 
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
+    Arg:
```

Comment:
Add newline above this.

Reply:
@kevinlee12 , hello i'm sorry i mistakenly committed main.py file with it, is there way i can deattach from here?? On basis of your comment actually i'm not sure here what you are asking me to do, please can you explore more it'll be a great help
Thank you

Reply:
Sure, you can keep this file here and do it as part of this PR. No harm done here. 

As for what I'm asking for, add an empty line between 114 and 115.

Reply:
Done